composer.json

```diff
@@ -4,50 +4,59 @@
     "description": "Shipping data export environment for Sylius platform applications",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
-
-        "sylius/sylius": "^1.4"
+        "php": "^7.3",
+        "sylius/sylius": "~1.8.0 || ~1.9.0"
```

Comment:
Why not just "^1.8"?

Reply:
Update to 1.9 and we are allowing 1.8 ?

Reply:
Update to 1.9 and You are allowing 1.8?

Reply:
That's cool if we gave opportunity to use also in 1.8

Reply:
The plugin will work as well with both versions. In GitHub Actions was made a matrix for 1.8 and 1.9 versions.